examples/resource_lacework_query/main.tf

```diff
@@ -0,0 +1,40 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+  resource "lacework_query" "example" {
+    query_id       = var.query_id
+    evaluator_id   = "Cloudtrail"
+    query          = var.query
```

Comment:
nit.  should we call this query_text to be consistent with Lacework APIs and SDK structs?